minigu/storage/tests/integration_tests.rs

```diff
@@ -0,0 +1,8 @@
+// Integration tests for minigu-storage
+
+mod ap;
+mod common;
+mod tp;
+
+pub use ap::*;
+pub use tp::*;
```

Comment:
There is no need to export `ap::*` and `tp::*`. Functions marked with `#[test]` can be identified automatically.